tests/test_indexer.py

```diff
@@ -140,11 +133,37 @@ def test_indexed_file_with_invalid_content_type(self):
         self.verify_index_document_structure_and_content(search_results[0], bundle_key,
                                                          files=smartseq2_paried_ends_indexed_file_list)
 
+    def test_debug_message_logged_when_indexing_an_invalid_bundle_name(self):
```

Comment:
In the long term, I don't know if this is a meaningful test.  I imagine errors would be logged to a centralized zone, and really you should be testing that the errors make it there.

Reply:
The current name of this test understates the purpose and importance of this test. All object creation events get passed to the indexer, and this checks processing of a non-bundle key. It ensures that no unexpected exceptions are thrown, that it is not added to Elasticsearch, ... and exercises/checks a log message to get the code coverage back into the passing range.
I propose changing the test name to be more representative, such as: test_nonbundle_key